vertical-pod-autoscaler/hack/deploy-for-e2e-locally.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+
+function print_help {
+  echo "ERROR! Usage: deploy-for-e2e-locally.sh [suite]*"
+  echo "<suite> should be one of:"
+  echo " - recommender"
+  echo " - recommender-externalmetrics"
+  echo "If component is not specified all above will be started."
```

Comment:
This doesn't seem to be true. If there is no component specified script prints help and exits